File: src/main/java/org/alham/alhamfirst/dto/todo/TodoDTO.java
package org.alham.alhamfirst.dto.todo;

import lombok.*;

import java.time.LocalDate;

@Getter
@Setter
@NoArgsConstructor
@Builder
@AllArgsConstructor
public class TodoDTO {

    private Long id;
    private String detail;
    private boolean completed;
    private LocalDate startDate;

}

File: src/main/java/org/alham/alhamfirst/dto/todo/TodoDTO.kt
package org.alham.alhamfirst.dto.todo;

import lombok.*;

import java.time.LocalDate;

//@Getter
//@Setter
//@NoArgsConstructor
//@Builder
//@AllArgsConstructor
data class TodoDTO(var id: Long? = null,
                   var userId : Long? = null,
                   var detail : String = "",
                   var completed : Boolean = false,
                   var startDate : LocalDate = LocalDate.now()) {

}
